taxi/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(Driver, related_name="cars")
```

Comment:
it's better to use `settings.AUTH_USER_MODEL`

Why gitignore in the .idea folder?) Add .idea to gitignore and remove unnecessary files

Reply:
have no idea how to remove .idea from pull request(

Reply:
https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository